nscplugin/src/main/scala/scala/scalanative/nscplugin/PrepNativeInterop.scala

```diff
@@ -5,12 +5,16 @@ import scala.tools.nsc
 import nsc._
 
 import scala.collection.immutable.ListMap
-import scala.collection.mutable
+import scala.collection.mutable.Buffer
 
+// Partially
```

Comment:
This comment is not very clear any longer. (i.e. what does partially mean here exactly?)

Lets put "ported from Scala.js" down to the list item `Rewrite calls to scala.Enumeration.Value`.